totest-manager.py

```diff
@@ -62,15 +63,31 @@ def __init__(self, project, dryrun=False, api_url=None, openqa_server='https://o
             test_subproject = 'ToTest'
         self.test_project = '%s:%s' % (self.project, test_subproject)
         self.openqa = OpenQA_Client(server=openqa_server)
-        self.issues_to_ignore = []
-        self.issuefile = "{}_{}".format(self.project, ISSUE_FILE)
-        if os.path.isfile(self.issuefile):
-            with open(self.issuefile, 'r') as f:
-                for line in f.readlines():
-                    self.issues_to_ignore.append(line.strip())
+        self.load_issues_to_ignore()
         self.project_base = project.split(':')[0]
         self.update_pinned_descr = False
 
+    def load_issues_to_ignore(self):
+        url = self.api.makeurl(['source', self.project, '_attribute', 'OSRT:IgnoredIssues'])
```

Comment:
Does this also exist in IBS? We are about to use ttm for SLE as well so I hope this does not break it

Reply:
it could exist - it doesn't yet